deploy/infrastructure/modules/terraform-aws-dss/README.md

```diff
@@ -0,0 +1,75 @@
+# terraform-aws-dss
+
+This terraform module creates a Kubernetes cluster in Amazon Web Services using the Elastic Kubernetes Service (EKS) 
+and generates the tanka files to deploy a DSS instance.
+
+
+## Getting started
+
+### Prerequisites
+
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Amazon Web Services](./README.md#amazon-web-services)
+
+    
+#### Amazon Web Services
+
+1. Install and initialize [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions).
+    1. Confirm successful installation with `aws --version`.
+2. If you don't have an account, sign-up: https://aws.amazon.com/free/
+3. Configure terraform to connect to AWS using your account.
+   1. We recommend to create an AWS_PROFILE using for instance `aws configure --profile aws-interuss-dss`
+      Before running `terraform` commands, run once in your shell: `export AWS_PROFILE=aws-interuss-dss`
+      Other methods are described here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration
+   
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named for instance `terraform-aws-dss-dev`.
```

Comment:
nit (I was thrown off by "instance"="example" rather than "instance"="deployed copy of software"):

```suggestion
1. Create a new folder in `/deploy/infrastructure/personal/` named, for instance, `terraform-aws-dss-dev`.
```

Reply:
fixed, thank you